src/locales/fr/PAGES/SUMMARY.json

```diff
@@ -0,0 +1,19 @@
+{
+    "TITLE": "Étape 5: Résultat du rapport de l’évaluation",
```

Comment:
```suggestion
    "TITLE": "Étape 5: Finaliser le rapport",
```

Reply:
@remibetin do you agree with that suggested change "Finaliser le rapport" ?
It reflects better the actual display of the page and starts with a verb as the other tabs